components/intake.py

```diff
@@ -18,16 +20,19 @@ def __init__(self):
             ids.PcmChannels.Intake.intake_piston_reverse,
         )
 
-    def execute(self):
+    def do_intake(self) -> None:
+        self.deployed = True
+
+    def end_intake(self) -> None:
```

Comment:
These are _very_ unusual names. I don't think it makes sense to tell an intake subsystem to "do intake" or "end intake".

Reply:
Should I change it to "intake" and "retract"?

Reply:
Is there a reason they were renamed from "deploy"/"retract", especially since the variable is still called "deployed"?

Reply:
I renamed it when writing a controller for the intake at the very start.
Now that controller doesn't exist anymore I will change it back to "deploy" and "retract", if that is better?